README.md

```diff
@@ -184,6 +184,8 @@ the first available will be chosen, but you can force a specific one:
 let test#python#runner = 'pytest'
 " or
 let test#python#runner = 'nose'
+" or
+let test#python#runner = 'django'
```

Comment:
Hmm, I'm wondering if moving Django in `g:test#runners` to be the first is a better choice. If `manage.py` undeniable identifies a Django project, it's better that in that case the Django runner is being run, isn't it? In that case we wouldn't need the `test#python#runner` conditional, because these doesn't need to be checked for Django. What do you think?